scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/job/InternalJob.java

```diff
@@ -1022,11 +1022,12 @@ public void start() {
         setNumberOfRunningTasks(0);
         setStatus(JobStatus.RUNNING);
 
-        HashMap<TaskId, TaskStatus> taskStatus = new HashMap<>();
+        List<InternalTask> tasks = getITasks();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename "tasks" which hides the field declared at line 101. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AHiddenFieldCheck)